schema.sql

```diff
@@ -6,119 +6,118 @@ CREATE DATABASE PiccadillyPizza;
 DROP TABLE IF EXISTS customer;
 CREATE TABLE customer (
   id SERIAL PRIMARY KEY,
-  name VARCHAR ( 255 ),
-  username VARCHAR ( 255 ) NULL,
-  password VARCHAR ( 255 ) NULL,
+  name VARCHAR ( 255 ) NOT NULL,
+  username VARCHAR ( 255 ) NOT NULL,
+  password VARCHAR ( 255 ) NOT NULL,
   phone_number INTEGER NULL
 );
 
 DROP TABLE IF EXISTS credit_card;
 CREATE TABLE credit_card (
   id SERIAL PRIMARY KEY,
-  customer_name VARCHAR ( 255 ) NULL,
-  card_number INTEGER NULL,
-  apartment_number VARCHAR ( 255 ),
-  street_address VARCHAR ( 255 ) NULL,
-  zipcode INTEGER,
-  customer_id INTEGER REFERENCES customer ( id ),
-  expiration_date TIMESTAMP NULL,
-  card_type VARCHAR ( 255 ),
-  city VARCHAR ( 255 ),
-  state VARCHAR ( 255 )
+  customer_name VARCHAR ( 255 ) NOT NULL,
+  card_number INTEGER NOT NULL,
+  billing_address VARCHAR ( 255 ) NOT NULL,
```

Comment:
Because a customer can have multiple addresses on file, but none of those addresses need to match to the billing address of the card on file. So as a customer I can deliver to Learners Guild and to my best friend's house, but my billing address for my credit card is my home address.